src/CommunityToolkit.Maui/Converters/ListToStringConverter.shared.cs

```diff
@@ -25,7 +27,9 @@ public override string ConvertFrom(IEnumerable value, Type targetType, object? p
 	{
 		ArgumentNullException.ThrowIfNull(value);
 
-		if ((parameter ?? Separator ?? string.Empty) is not string separator)
```

Comment:
Since `public string Separator { get; set; }` is not nullable, we can remove the unused `??`